### PR TITLE
Create Prestashop.com.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

**Comment by ekaris** [comment]
@J0WI Good to go

**Comment by J0WI** [comment]
WFM:
ambassadors.prestashop.com
validator.prestashop.com

Refused:
scm.prestashop.com
themes.prestashop.com

Mismatch:
status.prestashop.com

File: src/chrome/content/rules/Prestashop.com.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!--
+
+	non working:
```
Comment (J0WI) [inline src/chrome/content/rules/Prestashop.com.xml]: Please clarify why it's not working

**Comment by ekaris** [comment]
Done

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**